ublox-bluetooth-commands.go

```diff
@@ -147,36 +155,40 @@ func (btd *BluetoothDevices) MultiDiscoverWithContext(ctx context.Context, scant
 	// Don't concatenate errors. We need to spot the ContextCancelled error
 	err = <-errChan
 	if noOfDevices == 2 {
-		err = <-errChan
+		err2 := <-errChan
```

Comment:
could this cause a block in this thread?

Reply:
If we have two dongles, we will get 2 channels signalled

Reply:
True but the thread depends on both returning, there's always the possiblity of a timeout

Reply:
Code has been in there since dual dongle began and has seen no issue

Reply:
The go function that signals the error channel ensures it will always happen